openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -256,6 +256,15 @@ def __init__(self, **data):
         if any(len(pur_list) > 2 for pur_list in self.data.values()):
             raise NotImplementedError("Can't stitch together results yet")
 
+    @staticmethod
+    def compute_mean_estimate(dGs:list[dict]):
+        u = dGs[0].u
+        # convert all values to units of the first value, then take average of magnitude
+        # this would avoid a screwy case where each value was in different units
+        vals = [dG.to(u).m for dG in dGs]
+
+        return np.average(vals) * u
```

Comment:
Given that the two staticmethods are effectively the same thing except from this call, how about passing through a callable (i.e. the relevant numpy function) and just standardising it to a single method?

Reply:
I'm mostly suggesting this because it is kind of a temporary solution to a temporary CLI state.

Reply:
There's a lot I don't like my fix here, but I think it's the most clear in the short-term. If we pass through a callable, that means we would need to specify _which_ callable on the `gather.py` side, and I want to keep that all within the protocol.

Reply:
Yeah that's fair. I'll say for now that I don't really mind that much.

However I would, order ~ 2-3 months, would love it if we didn't depend on shims in the Protocol itself. It's fine for now, but when we (I'll say "I" because it's not so enthralling work and I'll probably spend my time on it after ABFEs) do the refactor it would be great to not be in a situation where we could go without the staticmethods. At least it's the first thing I'd likely axe in a feflow move refactor.

Reply:
I agree that this can be handled way better with a bit more refactoring, and that we unfortunately need a shorter-term fix than that. Merging for now and opened and issue here.

documenting here to make sure it gets addressed in the medium-term: https://github.com/OpenFreeEnergy/openfe/issues/1056